Snowdog_SolrNavigation/templates/filters.phtml

```diff
@@ -2,7 +2,13 @@
 /** @var \Snowdog\SolrNavigation\Block\Filters $block */
 $filters = $block->getFilterBlocks();
 ?>
-
+<button
+    class="button button button--secondary filters__skip-button"
+    type="button"
+    data-mage-init='{"skipFilters": {}}'
+>
+    <?= __('Skip to products') ?>
+</button>
 <?php if (!empty($filters)): ?>
     <h1 class="filters__heading">
```

Comment:
it shouldn't be `h1`. I know it's inside the sidebar so theoretically it passes the tests but not all AT supports it and it's better to use only one h1 on the page (as a page main title) and h2 for the rest of main sections' or navigation headings

Reply:
I know, just forgot to change this one heading :)

Reply:
👍